### DISCUSSION TITLE
Erro ao usar o select

### DISCUSSION THREAD

**Discussion by acsk** [balta-io/forum#216] [Tira Dúvidas] [answered]
O curso está numa vesão antiga, quando tento replicar o que estou estudando acaba gerando erros a qual não sei exatamente o que é como esse abaixo, o select não está conseguindo atribuir o 'cart' dá erro.
Estou usando
- Angular:13 
- Ionic: 6

`
import { CartModel } from './../../models/cart.model';
import { Component, OnInit } from '@angular/core';
import { Observable } from 'rxjs';
import { select, Store } from '@ngrx/store';

@Component({
  selector: 'app-navbar',
  templateUrl: './navbar.component.html',
  styleUrls: ['./navbar.component.scss'],
})
export class NavbarComponent implements OnInit {
  cart$!: Observable<CartModel>
  constructor(
    private store: Store<CartModel>
  ) {
    this.cart$ = store.pipe(select('cart')).  ///// ERRO NESSA    LINHA NO `SELECT`
  }

  ngOnInit() { }

}
`

**Comment by acsk** [comment, marked as answer]
O cruso é 
**"Criando Apps Web com Angular e Redux"**

**Reply by andrebaltieri** [reply]
@acsk é necessário utilizar as mesmas versões que estão nos cursos. Depois de terminar eles, aí você pode tentar versões mais novas por que mudam algumas coisas mesmo... Mas ai com mais experiência você consegue desenrolar.
